### DISCUSSION TITLE
How to add third party <body> scripts to Nuxt?

### DISCUSSION THREAD

**Discussion by mhabbasi-gh** [nuxt/nuxt#8939] [Questions (Nuxt 2)]
Hello!

I want to add third party script like this to my Nuxt code:

<script>
thirdParty.method({
            key: myKey,
            div: document.getElementById('thirdParty'),
            src: source.com,
        });
</script>

I can just throw it into my component's template body (This script has to be inside <**body**> not <**head**>) and interestingly enough it works but there are two issues here:

1. It looks ugly
2. It's not dynamic. For example I can't bind selector dynamically to a prop or a data property!

So I was wondering if anybody knows how can I integrate such scripts into my Nuxt project properly?

**Comment by udithishara** [comment]
You can make use of `plugins` to run custom JS code

https://nuxtjs.org/docs/2.x/directory-structure/plugins
https://stackoverflow.com/a/50140790

OR If you absolutely want, you can edit default html render template and add your script there

https://medium.com/dev-genius/customizing-the-nuxts-default-html-template-b2c6e768f2f7
